### ISSUE TITLE
Contao Backend Filter: Selectbox width too small

### ISSUE THREAD

**Issue by minstyle** [contao/contao#6208]
### Affected version(s)

4.13.26

### Description

In Contao, the widths of some selectboxes are too narrow and the whole content is not visible. Tested on Windows 11, Google Chrome 114 and Firefox 115.

Example of "System-Log" filter:

![grafik](https://github.com/contao/contao/assets/2649562/82999610-d5da-46a4-a543-d52aef76d391)

If I see this correctly, then inline CSS of `<div class="chzn-drop" style="left: 0px; width: 100%; top: 29px;">` causes the problem:

![grafik](https://github.com/contao/contao/assets/2649562/f684ea99-c01d-4b8b-b663-4f266b8c8cac)

When I remove `width: 100%;` in DOM via Chrome DevTools, then I can see the entire content.

![grafik](https://github.com/contao/contao/assets/2649562/3e1acf88-5fd3-408c-9ef7-37cd488c3718)

**Comment by cliffparnitzky** [comment]
Just to clarify, the field is a dropdown aka select ... not a checkbox.

**Comment by minstyle** [comment]
> Just to clarify, the field is a dropdown aka select ... not a checkbox.

Yes, you are right. Thanks for the hint. It was a long day at work today and I must have confused the terms. I have updated the issue.

**Comment by minstyle** [comment]
Also in Contao 4.13.30 the problem still exists. Would be nice if it could still be fixed.

**Comment by leofeyer** [comment]
I can confirm the issue, but removing the `width: 100%` makes the back end jump when opening and closing the drop-down menu. Can you check whether Chosen has an option to right-align the drop-down menu?

**Comment by zoglo** [comment]
See:
https://github.com/contao-components/chosen/blob/da03c67010b8fc520f64f2a9bbfcafb453ad2489/js/chosen.js#L72

Hardcoded and `chosen.js` hasn't been updated for 6 years.

Should we rewrite this on our own and implement that feature until we find a new solution? (changing chosen.js was mentioned at the CC 2023 so... - possible to `add a feature to fix that bug`)

**Comment by leofeyer** [comment]
There is a PR to replace Chosen with Choices (see #4363), but there are still CSS issues. Halfway through fixing these, we then thought about using [Tom Select](https://tom-select.js.org) instead. 🙈

**Comment by zoglo** [comment]
@leofeyer If this is considered a bug, would the PR with choices be a bugfix or considered a feature?
Might be worth it to look into the CSS Issues if the PR is already "done and easy to merge"

**Comment by zoglo** [comment]
Should not be the case with 5.5 anymore but would still be a bug in 5.3
